projects/demo/src/modules/app/pages.ts

```diff
@@ -955,19 +955,19 @@ export const pages = [
         section: `Customization`,
         title: `i18n`,
         keywords: `i18n, internationalization, local, translate, language`,
-        route: `i18n`,
+        route: `/i18n`,
     },
     {
         section: `Customization`,
         title: `Variables`,
         keywords: `colors, css, vars, custom, properties, style`,
-        route: `variables`,
+        route: `/variables`,
     },
     {
         section: `Customization`,
         title: `Wrapper`,
         keywords: `colors, css, theme, custom, style`,
-        route: `wrapper`,
+        route: `/wrapper`,
```

Comment:
can we encapsulate `/` prefix inside `tuiGoto`?

Reply:
`prefix` is useful not only inside `tuiGoto` utility.

\+ it is nice to have the same pattern for the whole file (all other routes starts their routing with `/`)